sqladmin/models.py

```diff
@@ -734,25 +734,27 @@ def get_attr_value(
             result = getattr(obj, attr.key)
             result = result.value if isinstance(result, Enum) else result
 
-        return result or ""
+        return result
```

Comment:
I found it also causes another bug. Is this an intentional change?

Reply:
What issue did you face with this?

Reply:
It returns `""` even if the `result` is an empty `InstrumentedList`, so it causes the following error:

```python
    def _url_for_details_with_attr(self, obj: Any, attr: RelationshipProperty) -> str:
        target = getattr(obj, attr.key)
        if target is None:
            return ""
    
>       pk = getattr(target, attr.mapper.primary_key[0].name)
E       AttributeError: 'InstrumentedList' object has no attribute 'id'
```